addon/templates/components/polaris-drop-zone/file-upload.hbs

```diff
@@ -13,11 +13,11 @@
 {{else if (eq size "large")}}
   {{#polaris-stack vertical=true spacing="tight"}}
     {{#if (eq type "file")}}
-      <img class={{imageClasses}} alt="" src={{assetFileUpload}}>
+      {{svg-jar assetFileUpload class=imageClasses}}
```

Comment:
any reason for rendering these as `svg` instead of `img`? Just asking cause this is what Polaris does and also if we can avoid `svg-jar` processing that's also a plus I think

Reply:
In its current form the images don't load in the consuming app because these assets aren't added to the host app's assets. This is why I started looking into this 😄 

I looked at removing svg-jar before making this change but it looks like we do it use it in other components within this app, so went with this.

Reply:
ahh, got it
I know this worked initially, but then moved this repo to `@smile-io` org, so that broke this
I think if we change `/ember-polaris/images/file-upload.svg` -> `/@smile-io/ember-polaris/images/file-upload.svg` should work....cause the addon's assets in `public` should be merged in the host app right?

Reply:
Its now at `@smile-io/ember-polaris/ember-polaris/images/file-upload.svg` 😞

Reply:
damn, ok....let's just use this branch for now